code/modules/ai/ai_holder_combat.dm

```diff
@@ -70,21 +70,22 @@
 /datum/ai_holder/proc/melee_attack(atom/A)
 	pre_melee_attack(A)
 	. = holder.IAttack(A)
-	if(.)
+	world.log << "melee_attack: [.]"
```

Comment:
Not sure if you meant to leave this here, but that could spam the log a lot.

Reply:
Whoops, removed